pages/get-involved.tsx

```diff
@@ -134,6 +141,61 @@ function PerfectFit() {
   );
 }
 
+function TestimonialSection() {
+  const theme = useTheme();
+  const isMdOrLarger = useMediaQuery(theme.breakpoints.up('md'));
+  const settings = {
+    dots: true,
+    infinite: true,
+    // autoplaySpeed: 5000,
+    // autoplay: true,
+    slidesToShow: isMdOrLarger ? 2 : 1,
+    slidesToScroll: 1,
+  };
+
+  return (
+    <Box>
+      {isMdOrLarger ? (
+
+        <Slider {...settings}>
+          {content.testimonies.map((testimony) => (
+            <Testimonials
+              key={testimony.id}
+              testimonial={testimony.testimonial}
+              imageSrc={testimony.imageSrc}
+              name={testimony.name}
+              cvrole={testimony.cvrole}
+            />
+          ))}
+        </Slider>
+
+      ) : (
```

Comment:
Let's add an extra line below this, to make all of the spacing even, but I appreciate these extra lines to break up the content for my eyes.

Reply:
done

Reply:
Thank you! Will you just run through this file and give a line of space in between each of the major function pieces? A few are missing it.

Reply:
done